### DISCUSSION TITLE
I have a problem with losetup

### DISCUSSION THREAD

**Discussion by dopaemon** [cirruslabs/cirrus-ci-docs#916] [Q&A] [answered]
are there any examples of using losetup and loopX device on current CirrusCI?
Thank CirrusCI Team

**Comment by fkorotkov** [comment, marked as answer]
Hey @dopaemon,

Could you please elaborate on what exactly are you trying to achieve? You should be able to use `losetup` in a [privileged container](https://cirrus-ci.org/guide/linux/#kvm-enabled-privileged-containers) or in a [custom VM](https://cirrus-ci.org/guide/custom-vms/).

**Reply by dopaemon** [reply]
i use this in privileged container

**Reply by dopaemon** [reply]
Thanks
